### ISSUE TITLE
`mix pow.install` throws error

### ISSUE THREAD

**Issue by iamreinder** [pow-auth/pow#725]
When I try to run `mix pow.install` it throws an error:
```
** (MatchError) no match of right hand side value: #Macro.Env<aliases: [{JS, Phoenix.LiveView.JS}], context: nil, context_modules: [PowResetPassword.Phoenix.ResetPasswordHTML]
```

I'm new to Elixir/Phoenix and I have no idea where to begin debugging this. I've searched within the issues here on GitHub, searched on Google and asked ChatGPT. But I haven't come any further solving the issue. Any idea what might be the solution?

I use Elixir 1.16.1 on Erlang 26.2.1 with Phoenix Framework 1.7.11.

**Comment by danschultzer** [comment]
Thanks! This was a issue introduced with the `:debug_heex_annotations` in Phoenix LiveView. I've released Pow v1.0.37 that resolves it, just upgrade and it should work for you now! For previous versions you'll need to set that setting to false or remove it entirely.

**Comment by iamreinder** [comment]
Thanks @danschultzer! Before upgrading, I tried the setting you mentioned. That indeed also worked. I'm still new to Elixir and Phoenix, so I'm trying to get an idea of how everything works together and how I can resolve issues that occur 😅
